apps/fluent-tester/src/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -54,25 +46,27 @@ const SizeCheckbox: React.FunctionComponent = () => {
 };
 
 const OtherCheckbox: React.FunctionComponent = () => {
-  const [isCheckedControlled1, setCheckedControlled1] = React.useState(false);
-  const onChangeControlled1 = React.useCallback((checked) => {
-    setCheckedControlled1(checked);
+  const [isChecked, setisChecked] = React.useState(false);
+
+  const setCheckedTrue = React.useCallback(() => {
+    setisChecked(true);
   }, []);
 
-  const [isCheckedControlled2, setCheckedControlled2] = React.useState(true);
-  const onChangeControlled2 = React.useCallback((checked) => {
-    setCheckedControlled2(checked);
+  const setCheckedFalse = React.useCallback(() => {
+    setisChecked(false);
   }, []);
 
   return (
-    <View>
-      <Checkbox label="This is a controlled Checkbox" onChange={onChangeControlled1} checked={Boolean(isCheckedControlled1)} />
-      <Checkbox
-        label="Checkbox rendered with labelPosition 'before' (controlled)"
-        onChange={onChangeControlled2}
-        labelPosition="before"
-        checked={Boolean(isCheckedControlled2)}
-      />
+    <View style={commonStyles.customisedChecked}>
```

Comment:
If this style is only used in this page, does it need to be in common? Could we just have a page wide stylesheet?

Reply:
Changes completed and approved , hence resolving.